src/OSPSuite.Core/Converters/v12/Converter110To120.cs

```diff
@@ -56,6 +58,17 @@ private void convertEventGroupBuildingBlock(XElement eventGroupNode)
          _converted = true;
       }
 
+      private void convertBuildMode(XElement parameterNode)
+      {
+         var buildMode = parameterNode.GetAttribute(_buildMode);
+         // This way around, a check for null is not necessary
+         if (!"Property".Equals(buildMode))
+            return;
```

Comment:
unusual, but nice :)

Reply:
I'm going to change it to `string.Equals` which has the same feature